source/reference/hyperlinks.txt

```diff
@@ -0,0 +1,102 @@
+.. _style-guide-hyperlinks:
+
+==========
+Hyperlinks
+==========
+
+You can link out to other sites, internal pages or specific sections on
+a page using inline hyperlinks and cross-referenced hyperlinks.
+
+Inline Hyperlinks
+-----------------
+
+Use inline hyperlinks when you want to link to a website with its full URL.
+
+Example
+~~~~~~~
+
+.. code-block:: rst
+
+   Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Rendered View
+`````````````
+
+Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Cross-Referenced Hyperlinks
+---------------------------
+
+Use cross-referenced hyperlinks when you want to link to another page in
+docs. This page can be in the same repo, or another within the ``MongoDB``
+organization.
```

Comment:
Nothing to fix here--just curious why "MongoDB" is in monospace.

Reply:
haha I'm not sure why I did -- certainly doesn't make sense to, so I'm removing it